src/bg/background.js

```diff
@@ -4,6 +4,9 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
     }
     else if ("getTheme" in request) {
         getTheme(sendResponse);
+    } 
+    else if ("deleteOne" in request && "id" in request) {
```

Comment:
Hi @zurda ,

Since there is now functionality for deleting one item

Maybe you can do additional refactor by:
- move the code of clear item and clear all to one function that is called from `popup.js` by 
`{clear: true timestamp: ..}` will clear specific item
`{clear:true}` will clear all items

and then in `popup.js` check if clear is in the request and handle it in its own function, and then remove clear code from `getClippings`